docker/qgis.dockerfile

```diff
@@ -8,6 +8,9 @@ ARG CACHE_DIR
 FROM  qgis/qgis3-build-deps:${DOCKER_TAG}
 MAINTAINER Denis Rouzaud <denis@opengis.ch>
 
+LABEL Description="Docker container with QGIS for CI testing" Vendor="QGIS.org" Version="1.1"
```

Comment:
for CI testing only?

Reply:
That's the purpose why it was created. What other use cases?
For regular tests there are the nightlies.

Reply:
Wouldn't it be possible to run QGIS from them on a desktop too? Just wondering...